presto-hudi/src/main/java/com/facebook/presto/hudi/HudiSessionProperties.java

```diff
@@ -107,6 +109,11 @@ public HudiSessionProperties(HiveClientConfig hiveClientConfig, HudiConfig hudiC
                         READ_MASKED_VALUE_ENABLED,
                         "Return null when access is denied for an encrypted parquet column",
                         hiveClientConfig.getReadNullMaskedParquetEncryptedValue(),
+                        false),
+                integerProperty(
+                        MAX_PARTITIONS_PER_SCAN,
```

Comment:
Why do we need this property?

Reply:
To limit partition like the implementation in hive connector.